src/map.config.json

```diff
@@ -0,0 +1,108 @@
+{
+  "mapcontainer": "mapContainer",
+  "startingViewPointName": "rennes",
```

Comment:
the api has changed now it's "startingViewpointName" instead of "startingViewPointName"